terraform_files/baremetal_infra_env/limit_ip_dhcp_range.xsl

```diff
@@ -9,6 +9,17 @@
      </xsl:copy>
   </xsl:template>
 
+  <xsl:template match="/network/ip/dhcp/range">
```

Comment:
No. For DHCP IPv4 there is no `family` section at all so we must not do any filtering

Reply:
I'm not sure about that, because I do see:
![Screenshot from 2022-06-26 10-31-48](https://user-images.githubusercontent.com/7492909/175804263-761f9230-9c3f-4500-b713-74315fc3f82d.png)

What I think happens is that ipv6 does apply for that template, but has no ``.254`` substring to be replaced
Just to be on the safe side I'd change it to ``match="/network/ip[@family='ipv4']/dhcp/range"`` (because either way it works)

Reply:
ACK, it probably depends on the specific combination of DHCP then. I do see in my local libvirt that if I only have IPv4, then `family` is not present

![2022-06-26-185748_826x117_scrot](https://user-images.githubusercontent.com/2862927/175825336-f588f069-932b-49c6-8e4e-81394af59839.png)

I guess we need to handle both scenarios

Reply:
Ok, I have applied the change you suggested. In the end this override is meant to be working together with the scripts in this repo and not with an arbitrary libvirt environment. Unless we spot that this repo creates libvirt networks without `family` stanza, let's go with as you suggested